gimli/test/main.c

```diff
@@ -4,12 +4,17 @@
 #include "test.h"
 
 void print_help(char *name) {
-  printf("Run tests for Gimli.\n");
+  printf("\nRun tests for Gimli.\n");
   printf("Usage: %s TESTYPE\n", name);
+  #ifdef AVX_SUPPORT
```

Comment:
I would just wrap the whole thing inside the `ifdef` and not mention `TESTTYPE` at all if there is only one option. i.e., when AVX2 is disabled. Do you think this is a good idea? Also, it would be nice to add a sentence that says "run all the tests for gimli" or something similar.